client/sites/overview/components/migration-overview/components/migration-started-difm.tsx

```diff
@@ -1,21 +1,13 @@
-import { globe, group, Icon, scheduled } from '@wordpress/icons';
+import { globe, group, Icon, scheduled, envelope } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
 import { Container, Header } from './layout';
-import type { SiteDetails } from '@automattic/data-stores';
 
-export const MigrationStartedDIFM = ( { site }: { site?: SiteDetails } ) => {
+export const MigrationStartedDIFM = () => {
 	const translate = useTranslate();
-	const migrationSourceSiteDomain = site?.options?.migration_source_site_domain
-		? site?.options?.migration_source_site_domain?.replace( /^https?:\/\/|\/+$/g, '' )
-		: translate( 'your site' );
 
-	const title = translate( 'Your migration is underway' );
+	const title = translate( "We've received your migration request" );
 	const subTitle = translate(
-		"Sit back as {{strong}}%(siteName)s{{/strong}} transfers to its new home. Here's what you can expect.",
-		{
-			components: { strong: <strong /> },
-			args: { siteName: migrationSourceSiteDomain },
-		}
+		"Our team has received your shared details and will review your site to make sure we have everything we need. Here's what you can expect next!"
```

Comment:
Shared details feels a little redundant. Can we simplify this to:

> Our team has received your details. We will review your site to make sure we have everything we need. Here's what you can expect next:

Reply:
Done!